### PR TITLE
[Berk] Run Intg tests on compatible branch

### PR DESCRIPTION
**Pull request by dkijania**

### CODE REVIEW

File: genesis_ledgers/mainnet.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
   "genesis": {
     "genesis_state_timestamp": "2021-03-17T00:00:00Z"
   },
+  "daemon": {
```
Comment (mrmr1993) [inline genesis_ledgers/mainnet.json:5]: I don't think we want to include this in the next release, in case we accidentally ship it. Can you revert?

**Comment by dkijania** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Review by SanabriaRusso — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by nholland94** [comment]
!approved-for-mainnet

**Pull request merged**